From 7d18a77dfbef8ffe2b17b31098c75316b13f1597 Mon Sep 17 00:00:00 2001
From: Gyorgy Sarvari <skandigraun@gmail.com>
Date: Wed, 24 Nov 2021 21:10:56 +0100
Subject: [PATCH] xvnc: adapt for 1.21

From: Joakim Tjernlund <joakim.tjernlund@infinera.com>

VENDOR_STRING is gone as is DEFAULT_LOG_VERBOSITY
site.h has been removed and is not needed.

Upstream-Status: Backport [https://github.com/TigerVNC/tigervnc/commit/f2577107f7f55382c524d8c738a777e5cdd80f60]
Signed-off-by: Gyorgy Sarvari <skandigraun@gmail.com>
---
 unix/xserver/hw/vnc/xvnc.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/unix/xserver/hw/vnc/xvnc.c b/unix/xserver/hw/vnc/xvnc.c
index fda6205f..d16a4355 100644
--- a/unix/xserver/hw/vnc/xvnc.c
+++ b/unix/xserver/hw/vnc/xvnc.c
@@ -80,7 +80,6 @@ from the X Consortium.
 #undef VENDOR_RELEASE
 #undef VENDOR_STRING
 #include "version-config.h"
-#include "site.h"
 
 #define XVNCVERSION "TigerVNC 1.11.0"
 #define XVNCCOPYRIGHT ("Copyright (C) 1999-2020 TigerVNC Team and many others (see README.rst)\n" \
@@ -148,15 +147,14 @@ static Bool Render = TRUE;
 static Bool displaySpecified = FALSE;
 static char displayNumStr[16];
 
-static int vncVerbose = DEFAULT_LOG_VERBOSITY;
+static int vncVerbose = 0;
 
 
 static void
 vncPrintBanner(void)
 {
     ErrorF("\nXvnc %s - built %s\n%s", XVNCVERSION, buildtime, XVNCCOPYRIGHT);
-    ErrorF("Underlying X server release %d, %s\n\n", VENDOR_RELEASE,
-           VENDOR_STRING);
+    ErrorF("Underlying X server release %d\n\n", VENDOR_RELEASE);
 }
 
 static void
